client/pkg/connectionctx/manager_test.go

```diff
@@ -24,31 +24,31 @@ import (
 
 func TestManager(t *testing.T) {
 	re := require.New(t)
-	ctx := context.Background()
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
How about adding a test case that garbage collects one of the streams and verifies whether its context is canceled as expected?

Reply:
OK，I will add one

Reply:
done